### ISSUE TITLE
[BUG] Won't log in to Gamestop

### ISSUE THREAD

**Issue by Primetime146** [Strip3s/PhoenixBot#109]
# Expected Behavior
Open browser window and log into Gamestop

# Actual Behaviour

Browser window opens, bot says logging in, never logs in

![Capture](https://user-images.githubusercontent.com/80478049/114943099-7afe8580-9e13-11eb-8fbe-6b20c3d54e17.PNG)

# Desktop Configuration
Windows 10 Professional 
Chrome
Version 89

**Comment by Primetime146** [comment]
![Capture](https://user-images.githubusercontent.com/80478049/114943067-733ee100-9e13-11eb-808a-b83649730152.PNG)

**Comment by coolio79** [comment]
The same thing is happening to me, please fix! :)

**Comment by pyrometheous** [comment]
I fixed this today. Changed url on line 59 to:
`self.browser.get("https://www.gamestop.com/?openLoginModal=accountModal")`
Changed the wait to a sleep, and changed some buttons into xpaths instead of finding element by link test. Modded script attached:
[gamestop.zip](https://github.com/Strip3s/PhoenixBot/files/6517988/gamestop.zip)
So far it appears to work. I haven't had it actually check out yet. It also does not verify the max price, it'll just add it to the cart if it's in stock.
